scripts/lc-builds/blueos_clang-upstream-2018.12.03_omptarget.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+##
+## Copyright (c) 2017-19, Lawrence Livermore National Security, LLC.
+##
+## Produced at the Lawrence Livermore National Laboratory.
+##
+## LLNL-CODE-738930
+##
+## All rights reserved.
+##
+## This file is part of the RAJA Performance Suite.
+##
+## For details about use and distribution, please read RAJAPerf/LICENSE.
+##
+
+BUILD_SUFFIX=lc_blueos_clang-upstream-2018.12.03_omptarget
+RAJA_HOSTCONFIG=../scripts/lc-builds/clang_upstream_2018_12_03.cmake
+
+rm -rf build_${BUILD_SUFFIX} >/dev/null
+mkdir build_${BUILD_SUFFIX} && cd build_${BUILD_SUFFIX}
+
+module load cmake/3.9.2
+
+## NOTE: RAJA tests are turned off due to compilation issues.
+
+cmake \
+  -DCMAKE_BUILD_TYPE=Release \
+  -C ${RAJA_HOSTCONFIG} \
+  -DENABLE_OPENMP=On \
+  -DENABLE_CUDA=Off \
+  -DENABLE_TARGET_OPENMP=On \
+  -DOpenMP_CXX_FLAGS="-fopenmp -fopenmp-targets=nvptx64-nvidia-cuda -fopenmp-implicit-declare-target" \
```

Comment:
FYI, this will not work once we merge this RAJA PR: https://github.com/LLNL/RAJA/pull/576

Reply:
Sorry I should have been more clear. This is to reproduce the clang-upstream error for the IBM folks. The purpose of this entire branch is to serve as a stable branch for bug reproduction for them. I don't think we will merge any of this into "develop" at any point.

Reply:
Now I realize I didn't need a PR for this . . .